enterprise/admin/views.py

```diff
@@ -328,7 +362,16 @@ def post(self, request, customer_uuid):
             if course_details:
                 course_mode = manage_learners_form.cleaned_data[ManageLearnersForm.Fields.COURSE_MODE]
                 course_id = course_details['course_id']
-                self._enroll_users(enterprise_customer, linked_learners, course_id, course_mode, request)
+                notification_type = manage_learners_form.cleaned_data.get(ManageLearnersForm.Fields.NOTIFY)
+                notify = notification_type == ManageLearnersForm.NotificationTypes.BY_EMAIL
```

Comment:
Are we expecting other types of notification in the future? Because in the form, you've defined the notification options as an enumerated choice set even though it's essentially a boolean choice, and in this part of the code you are explicitly treating it as a boolean.